frontend/apps/reader/modules/readerzooming.lua

```diff
@@ -26,6 +46,10 @@ local ReaderZooming = InputContainer:new{
         contentheight = _("Zoom to fit content height works best with page view."),
         content = _("Zoom to fit content works best with page view."),
     },
+    panned_modes = {
+        column = _("Page view normally works best with column zoom mode."),
+        pan = _("Pan zoom only works in page view."),
```

Comment:
Better to just change it.

Reply:
Sorry, I don't understand the point.

Reply:
In fact, I distinguished between both, because I see at least one case where column mode could be used with scrolling (if you want to read only one column in a text with parallel translation), while horizontal pan doesn't work at all in scroll mode.

Reply:
I mean this is a useless message. The "works best" messages are somewhat debatable, some long-time users don't necessarily agree, but if "only works" is accurate it's a message that should not exist. Besides the user experience, also keep in mind that there are more than two dozen volunteers who translate every string.

Reply:
OK; I understood change *the message*, you meant change *the setting*; so I'll do it.

Reply:
As `Pan mode` seems to be replacing `column mode`, I eventually removed the `panned_modes` table, and kept only one message. But for the reason I mentioned (reading only one of two parallel columns), I didn't automatically disable `scroll`.